aggregator/counter_elem.gen.go

```diff
@@ -179,13 +175,15 @@ func (e *CounterElem) AddUnique(timestamp time.Time, value float64, sourceID []b
 		lockedAgg.Unlock()
 		return errAggregationClosed
 	}
-	sourceHash := hash.Murmur3Hash128(sourceID)
-	if v, exists := lockedAgg.sourcesSeen[sourceHash]; exists && v == alignedStart {
+	source := uint(sourceID)
```

Comment:
Using a bitset is only beneficial when the maxSourceID is small? seems it will be simpler in the logic to just use a map?

Reply:
It depends on the max source id it received, but either way probably the memory difference is insignificant since the source set is cached. Map is also difficult to reuse as you need to delete the keys from the map to create an empty map, which is probably more overhead than just clearing bits.